mscore/exportdialog.cpp

```diff
@@ -446,10 +446,10 @@ void ExportDialog::accept()
       QString name = QString("%1/%2").arg(saveDirectory, cs->masterScore()->fileInfo()->completeBaseName());
       if (oneScore) {
             Score* score = scores.first();
-            name.append(QString(" - %1").arg(score->isMaster() ? tr("Full Score") : score->title()));
+            name.append(QString("%1").arg(score->isMaster() ? "" : "-" + score->title()));
```

Comment:
Not sure when this part is used, didn't notice a different behavoir when I had this part commented out?